pkg/reporter/reporter_test.go

```diff
@@ -255,11 +226,82 @@ var _ = PDescribe("Reporter", func() {
 
 		close(done)
 	}, 20)
+
+	Describe("AdditionalFields logic", func() {
```

Comment:
Don't need a describe here, should be top level.

Reply:
Ok